assets/package.json

```diff
@@ -72,6 +72,7 @@
     "file-loader": "^6.2.0",
     "fishery": "^2.2.2",
     "jest": "^26.6.3",
+    "jest-fail-on-console": "^3.0.2",
```

Comment:
note(non-blocking): It looks like [`3.1.0`](https://www.npmjs.com/package/jest-fail-on-console/v/3.1.0) was posted 4 days ago! can we use that?

Reply:
Thanks for noticing, just updated!